cppapi/client/DeviceAttribute.h

```diff
@@ -1318,6 +1318,9 @@ protected :
 #else
 	DeviceAttributeExt	            *ext;		// Class extension
 #endif
+
+private:
+    void init_common_class_members(const char * name,int dim_x,int dim_y);
```

Comment:
I would call it simply *init*. As it is so widely used it will never give any ambiguous and much shorter.

Reply:
I called it like that since it does not initialize everything.
ShortSeq is indeed a member.
I agree with you it would make sense to rename it to short_seq, as well as all the other similar members (DoubleSeq, ...).
My CLion instance does not seem to help me much in this task ("Refactor -> Rename" renames it only in DeviceAttribute.h. This is not very helpful), I don't know why...

About the Holder class, this is something that could be investigated in the future but I think there are more urgent tasks right now.